src/games/supported/Freeway.cpp

```diff
@@ -95,3 +98,38 @@ void FreewaySettings::loadState(Deserializer & ser) {
   m_terminal = ser.getBool();
 }
 
+// returns a list of mode that the game can be played in
+ModeVect FreewaySettings::getAvailableModes() {
+    ModeVect modes(8);
+    for(unsigned i = 0; i < 8; i++) {

```

Comment:
unsigned int. If only this was Python!

```return [ i for i in range(8) ]```

Also space between for and (.

Also, tie for the loop to sizeof(modes) or equivalent?

Reply:
Fixed.